reviewrot/gerritstack.py

```diff
@@ -90,34 +90,69 @@ def check_host_url(self, ssl_verify):
             if response.status_code == 200:
                 return True
             else:
-                raise ValueError('Host URL is incorrectly configured'
-                                 ' in config file.')
+                raise ValueError()
         except (ValueError, requests.ConnectionError):
```

Comment:
ConnectionError doesn't have to be related to incorrect URL in a config file. For example, DNS failure is also included in the exception and it can be confusing for users.

Reply:
I suggest removing this method, and just using default requests error handling with  requests.raise_for_status(), would you agree?

Reply:
@Allda

Reply:
The solution looks good.